samples/apps/copilot-chat-app/webapi/Program.cs

```diff
@@ -30,6 +31,9 @@ public static async Task Main(string[] args)
             .AddSemanticKernelServices()
             .AddPersistentChatStore();
 
+        // Add SignalR as the real time relay service
+        builder.Services.AddSignalR();
```

Comment:
Could we add this line to the rest of them below? Example:
```
builder.Services
    .AddApplicationInsightsTelemetry()
    .AddLogging(logBuilder => logBuilder.AddApplicationInsights())
    .AddAuthorization(builder.Configuration)
    .AddEndpointsApiExplorer()
    .AddSwaggerGen()
    .AddCors()
    .AddControllers()
    .AddSinglarR();
```

Reply:
The compiler doesn't like it when i try that.

Error CS1929 'ISignalRServerBuilder' does not contain a definition for 'AddApplicationInsightsTelemetry' and the best extension method overload 'ApplicationInsightsExtensions.AddApplicationInsightsTelemetry(IServiceCollection)' requires a receiver of type 'Microsoft.Extensions.DependencyInjection.IServiceCollection'

The current approach is easier than finding a work around and the way the tutorials on signalR recommend as well.